DTML.EduBot/Web.config

```diff
@@ -5,6 +5,10 @@
   -->
 <configuration>
   <appSettings file="Web.Secrets.Config">
+    <add key="tokenEndpoint" value="https://api.cognitive.microsoft.com/sts/v1.0/issueToken"/>
```

Comment:
You can move these to Web.Secrets.Config. I have pinned it to slack channel. And since that file is not tracked, it will never be checked in.